src/hooks/useBids.ts

```diff
@@ -49,6 +49,7 @@ export function useBids(saleId: string, saleType: SaleType): UseBidsReturn {
       {
         buyer: account!,
         amount: reduceTotalAmount,
+        status: bids.length > 0 ? bids[0].status : null,
       },
     ]
```

Comment:
The function name doesn't describe the return value. It should be clear what the function does.